CLAUDE.md

```diff
@@ -103,5 +103,4 @@ Read [cursor rules](.cursorrules).
 
 The frontend is contained in the site folder.
 
-For building Frontend refer to [this document](docs/contributing/frontend.md)
```

Comment:
Drive-by: link is dead.

Reply:
it's in docs/about/contributing/ now. can you leave this as is and I'll do a quick pr to fix the link?